windows/src/desktop/Makefile

```diff
@@ -26,13 +26,19 @@ help:
 insthelp:
   cd $(ROOT)\src\desktop\insthelp
   $(MAKE) $(TARGET)
-  
+
 inst:
   cd $(ROOT)\src\desktop\inst
   $(MAKE) $(TARGET)
 
+# ----------------------------------------------------------------------
+
+clean-desktop:
+    cd $(ROOT)\src\desktop
+    -del version.txt
```

Comment:
Does `version.rc` also need to be cleaned in the various Makefiles?

Reply:
Yes, will add to the def-clean target.